src/lib/components/navigation.ts

```diff
@@ -22,6 +23,16 @@ export const navigation: NavItem[] = [
       { name: "Signing", route: "signatures" },
     ],
   },
+  {
+    name: "Guides",
+    icon: "dataspace-icon",
```

Comment:
I think this should be switched to `guides-icon`, see other comment about this in `src/lib/components/Sidebar.svelte`.